src/main/kotlin/net/atos/zac/shared/exception/FoutMeldingExceptionMapper.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.shared.exception
+
+import jakarta.ws.rs.core.Response
+import jakarta.ws.rs.ext.ExceptionMapper
+import jakarta.ws.rs.ext.Provider
+
+/**
+ * Exceptionmapper om [FoutmeldingException] die gegooit zijn op te vangen en om te zetten
```

Comment:
Could translate this to EN? At some we need to look at this exception mapper btw since we have other similar exception mapping solutions in ZAC.. Should consolidate that I guess.

Reply:
Translated